### PR TITLE
crio: add periodic job for node-e2e

### PR DESCRIPTION
**Pull request by sohankunkerkar**

### CODE REVIEW

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodefeature-periodic
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodeconformance-periodic
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-conformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodeconformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-conformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodefeature-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodefeature-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-conformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodeconformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-conformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodeconformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodefeature-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodefeature-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodefeature-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodefeature-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodeconformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-conformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodefeature-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-conformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodeconformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodeconformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-conformance-periodic

**Comment by sohankunkerkar** [comment]
@rphillips could you please take a look at it?

**Comment by sohankunkerkar** [comment]
Previous run results: https://prow.ci.openshift.org/pr-history/?org=openshift&repo=release&pr=55802

File: ci-operator/step-registry/cri-o/node/e2e/conformance/test/cri-o-node-e2e-conformance-test-commands.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        daemon_reload: yes
+
+    - name: Disable selinux during e2e tests
+      command: 'setenforce 0'
+
+    - name: Run Conformance tests
+      become: yes
+      shell: |
+        export CONTAINER_RUNTIME_ENDPOINT="unix:///var/run/crio/crio.sock" &&\
+        make test-e2e-node FOCUS="\[Conformance\]" SKIP="\[sig-node\]\s*Pods\s*should\s*(delete\s*a\s*collection\s*of\s*pods|patch\s*a\s*pod\s*status|run\s*through\s*the\s*lifecycle\s*of\s*Pods\s*and\s*PodStatus)\s*\[Conformance\]" TEST_ARGS="--kubelet-flags='--cgroup-driver=systemd --cgroup-root=/ --cgroups-per-qos=true --runtime-cgroups=/system.slice/crio.service --kubelet-cgroups=/system.slice/kubelet.service'"
```
Comment (sohankunkerkar) [inline ci-operator/step-registry/cri-o/node/e2e/conformance/test/cri-o-node-e2e-conformance-test-commands.sh:43]: I'm skipping these tests as they are failing for cri-o + runc as well, so not related to crun

File: ci-operator/step-registry/cri-o/gather/cri-o-gather-commands.sh [hunk 2]
```diff
@@ -21,6 +22,7 @@ function getlogs() {
       fi
     done
     scp -rv "${SSHOPTS[@]}" "${USER}@${IP}:/tmp/artifacts/*" "${ARTIFACT_DIR}"
+    scp -rv "${SSHOPTS[@]}" "${USER}@${IP}:/tmp/_artifacts/"* "${ARTIFACT_DIR}/"
```
Comment (rphillips) [inline ci-operator/step-registry/cri-o/gather/cri-o-gather-commands.sh:25]: star isn't in the right spot?

Comment (sohankunkerkar) [reply ci-operator/step-registry/cri-o/gather/cri-o-gather-commands.sh:25]: whoops

Comment (sohankunkerkar) [reply ci-operator/step-registry/cri-o/gather/cri-o-gather-commands.sh]: Fixed

File: ci-operator/step-registry/cri-o/node/e2e/nodeconformance/test/cri-o-node-e2e-nodeconformance-test-commands.sh [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    - name: enable and start CRI-O
+      become: yes
+      systemd:
+        name: crio
+        state: started
+        enabled: yes
+        daemon_reload: yes
+    
+    - name: Disable selinux during e2e tests
+      command: 'setenforce 0'
```
Comment (haircommander) [inline ci-operator/step-registry/cri-o/node/e2e/nodeconformance/test/cri-o-node-e2e-nodeconformance-test-commands.sh:37]: also an issue for this, we should fix node e2e to work with selinux

**Comment by haircommander** [comment]
/approve

LGTM, please open issues in upstream corresponding to skipped tests so we can track

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodeconformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-conformance-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodeconfeature-periodic

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-crio-node-e2e-nodefeature-periodic

File: ci-operator/step-registry/cri-o/node/e2e/nodeconformance/test/cri-o-node-e2e-nodeconformance-test-commands.sh (hunk 3 above)
Comment (sohankunkerkar) [reply ci-operator/step-registry/cri-o/node/e2e/nodeconformance/test/cri-o-node-e2e-nodeconformance-test-commands.sh]: Fixed it

**Pull request merged**